bridge/client/app/_components/ktb-root-events-list/ktb-root-events-list.component.spec.ts

```diff
@@ -37,20 +37,18 @@ describe('KtbEventsListComponent', () => {
           },
         },
       ],
-    })
-      .compileComponents()
-      .then(() => {
-        fixture = TestBed.createComponent(KtbRootEventsListComponent);
-        component = fixture.componentInstance;
-        dataService = fixture.debugElement.injector.get(DataService);
-        dataService.loadProjects(); // reset project.sequences
-        // @ts-ignore
-        dataService.getProject(projectName).subscribe((pr: Project) => {
-          project = pr;
-          fixture.detectChanges();
-        });
-      });
-  }));
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(KtbRootEventsListComponent);
+    component = fixture.componentInstance;
+    dataService = fixture.debugElement.injector.get(DataService);
+    dataService.loadProjects(); // reset project.sequences
+    // @ts-ignore
+    dataService.getProject(projectName).subscribe((pr: Project) => {
```

Comment:
This one could eventually be problematic, because `beforeEach` does not wait for its completion, but it should be fine in the test. Could be changed to `project = await dataService.getProject(projectName).toPromise()` or we use the `done` parameter of `beforeEach` and add it inside the `subscribe`

Reply:
Added `done`